### PR TITLE
fix: APP-477 various Terrasos design tweaks

### PR DESCRIPTION
**Pull request by flagrede**
## Description

Closes: APP-477

Various Terrasos design tweaks:
- grid margins
- checkbox filters layout
- uppercase filter expand button
- read more project story color
- move stakeholders block and adjust spacing
- change responsive slider buttons color
- hide stakeholders if no data
- documentation table colors
- spanish translations tweaks
- terrasos logo cut off

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by flagrede** [comment]
@erikalogie You can check out the various design tweaks here:
Projects: https://deploy-preview-2551--terrasos.netlify.app/projects/1
Project page: https://deploy-preview-2551--terrasos.netlify.app/project/C02-009

File: web-components/src/components/organisms/ProjectFilters/ProjectFilters.tsx [hunk 1]
```diff
@@ -111,13 +111,11 @@ export default function ProjectFilters({
...
                 onClick={() => handleExpand(index)}
                 sx={{ fontWeight: '700' }}
-                className="text-sc-text-link"
+                className="text-sc-text-link uppercase"
               >
                 {isExpanded[index] ? labels.collapse : labels.expand}
               </ButtonBase>
             )}
-
-            <Divider sx={{ my: 5 }} />
```
Comment (blushi) [inline web-components/src/components/organisms/ProjectFilters/ProjectFilters.tsx:120]: it should only be removed for the last filter section, not for all see design https://www.figma.com/design/brkxGV5qNOkZUp0YQl1cxO/Terrasos-Phase-1?node-id=1204-174906&node-type=instance&m=dev

**Review by blushi — COMMENTED** [review]
Looks good. Just one comment and some missing translations:

image

**Comment by S4mmyb** [comment]
Thanks @flagrede ! I'm noticing the logo is still a bit cut off in both the english and spanish versions. Also, and sorry i missed this one before, it looks like we have another small language tweak on the header. "Home" should actually translate to "INICIO" rather than "HOGAR" ([link to line here](https://github.com/regen-network/regen-web/blob/e2cecf7f6e9e04a77ced3bec01b3be5a33b9c775/web-marketplace/src/lib/i18n/locales/es.po#L1961)). I think the translation worked correctly, but they just use the word start for the homepage instead - probably a regional thing.

**Review by blushi — APPROVED** [review]
LGTM

**Comment by flagrede** [comment]
@S4mmyb The logo should be good now. It was a bit hard to get the full size while matching the exact position between the two sites since we're using an SVG and they're using several PNGs.

**Comment by flagrede** [comment]
@erikalogie No problem, I have handled that in the PR.

**Pull request merged**